spec/openapi.yaml

```diff
@@ -68,6 +81,124 @@ components:
         - text
         - firmware
 
+    FileInfo:
+      type: object
+      properties:
+        type:
+          type: string
+          example: "PRINT_FILE"
+        refs:
+          type: object
+          properties:
+            download:
+              type: string
+              example: "/api/files/local/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode/raw"
+            icon:
+              type: string
+              example: "/api/thumbnails/local/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode.small.png"
+            thumbnail:
+              type: string
+              example: "/api/thumbnails/local/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode.orig.png"
+        size:
+          type: integer
+          example: 424242
+        path:
+          type: string
+          example: "/local/Spice_Harvester_0.3mm_PLA_MK3S_42m.gcode"
+        name:
+          type: string
+          example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+        creation_timestamp:
+          type: integer
+          example: 1648038843
+        ro:
+          type: boolean
+          example: False
+        metadata:
+          type: object
+          properties:
+            bed_temperature:
+              type: integer
+              example: 60
+            nozzle_temperature:
+              type: integer
+              example: 215
+            brim_width:
+              type: number
+              example: 0
+            estimated printing time (normal mode):
+              type: string
+              example: "42m 42s"
+            faded_layers:
+              type: integer
+              example: 42
+            filament cost:
+              type: number
+              example: 0.57
+            filament used [cm3]:
+              type: number
+              example: 12.44
+            filament used [g]:
+              type: number
+              example: 15.79
+            filament used [mm]:
+              type: number
+              example: 5170.06
+            filament_type:
+              type: string
+              example: "PLA"
+            filament_length:
+              type: number
+              example: 4.2
```

Comment:
How does `filament_length` relate to `filament used [mm]`? And what are these units?